OutOfSchool/OutOfSchool.WebApi/Services/ProviderService.cs

```diff
@@ -484,6 +484,11 @@ private protected async Task<ProviderDto> UpdateProviderWithActionBeforeSavingCh
             throw new ArgumentNullException(nameof(userId));
         }
 
+        if (await ExistsAnotherProviderWithTheSameEdrpouIpn(providerUpdateDto))
+        {
+            return null;
```

Comment:
Shouldn't any trace log be added here?

Reply:
Ok, I agree with you. I'll add logger.LogInformatiom() to this snippet of the code.